nettacker/api/engine.py

```diff
@@ -191,6 +193,37 @@ def index():
     )
 
 
+def sanitize_report_path_filename(report_path_filename):
+    """
+    sanitize the report_path_filename
+
+    Args:
+        report_path_filename: the report path filename
+
+    Returns:
+        the sanitized report path filename
+    """
+    filename = secure_filename(os.path.basename(report_path_filename))
+    if not filename:
+        return False
+    if "." in filename:
+        if (
+            filename.endswith(".html")
```

Comment:
Repetitive calls to filename.endswith() - reduces maintainability and extensibility. Suggested improvement: 

```

# Define a list or tuple of valid extensions
VALID_EXTENSIONS = (".html", ".htm", ".txt", ".json", ".csv")

# Check if the filename has a dot and ends with a valid extension
if "." in filename and filename.endswith(VALID_EXTENSIONS):


```

Reply:
Hi @securestep9  I've resolved the issue by implementing a fix as you suggested, Now i'm checking if there is a '.' in the filename if there is one then we look if the extension of that file is valid, I'm also accepting filenames with no extensions also because /core/graph.py has a condition which handles filenames which don't have any extensions.